spec/src/main/asciidoc/servlet-spec-body.adoc

```diff
@@ -9097,1024 +9097,14 @@ for Dependency Injection" of the JakartaEE Platform 9 specification.
-starts with “/” (15.1.3.2)
-
-Fix the inconsistency between this
-specification and HttpServletRequest.getPathInfo - the return value
-starts with “/” (15.1.3.2)
-
-Fix the inconsistency between this
-specification and HttpServletRequest.getPathTranslated - add the case
-when the container cannot translate the path (15.1.3.2)
-
-Allow HttpServletRequest.getAuthType to
-return not only pre-defined four authentication scheme but also the
-container-specific scheme (15.1.3.2)
-
-Change the behavior of
-ttpSessionListener.sessionDestroyed to notify before the session is
-invalidated (15.1.14.1)
-
-Fix the wrong status code of 403 to 404 (9.5,
-9.6)
-
-Element “taglib” should be “jsp-config”
-(13.2)
-
-Fix the version number of JSP specification
-to 2.0
-
-Fix the wrong formats (5.5, 6.2.5, 12.8.3,
-12.9)
-
-HTTP/1.1 is now required (1.2)
-
-<url-pattern> in <web-resource-collection> is
-mandatory (13.4)
-
-Clarification of IllegalArgumentException in
-the distributed environments (7.7.2)
-
-Clarification of error page handling (9.9.1,
-9.9.2, 9.9.3, 6.2.5)
-
-Clarification of Security Constraints,
-especially in the case of overlapping constraints (12.8)
-
-Clarification of the case when
-<session-timeout> element is not specified (13.4)
-
-Clarification of the case when the resource
-is permanently unavailable (2.3.3.2)
-
-Add missing getParameterMap() in the
-enumerated list (4.1)
-
-Clarification of the status code when
-/WEB-INF/ resource is accessed (9.5)
-
-Clarification of the status code when
-/META-INF/ resource is accessed (9.6)
-
-Change xsd:string to j2ee:string in
-deployment descriptor (13.4)
-
-Extensibility of deployment descriptors
-(SRV.13)
-
-XML Schema definition of deployment
-descriptor (SRV.13)
-
-Request listeners (SRV.10 and API change) +
-New API: ServletRequestListener, ServletRequestAttributeListener and
-associated event classes
-
-Ability to use Filters under the Request
-Dispatcher (6.2.5)
-
-Required class loader extension mechanism
-(9.7.1)
-
-Listener exception handling (10.6)
-
-Listener order vs. servlet init()/destroy()
-clarification (ServletContextListener javadoc change)
-
-Servlets mapped to WEB-INF / response
-handling (9.5)
-
-Request dispatcher / path matching rules
-(8.1)
-
-Welcome files can be servlets (9.10)
-
-Internationalization enhancements (5.4,
-14,2,22, 15.1.5)
-
-SC_FOUND(302) addition (15.1.5)
-
-“Relative path” in getRequestDispatcher()
-must be relative against the current servlet (8.1)
-
-Bug fix in the example of XML (13.7.2)
-
-Clarification of access by getResource “only
-to the resource” (3.5)
-
-Clarification of SERVER_NAME and SERVER_PORT
-in getServerName() and getServerPort() (14.2.16)
-
-Clarification: “run-as” identity must apply
-to all calls from a servlet including init() and destroy() (12.7)
-
-Login/logout description and methods added
-(12.10, 15.1.7)
-
+=== [[a384]] Compatibility with Jakarta Servlet Specification Version 4.0
```

Comment:
@gregw , please let me know what you thing about this solution.
 
Also I tried to find the rules for the anchor numbers but I didn't find them. Thus, if I need to change the numbers could you explain me how they are chosen?

Reply:
I think we can be stronger in this description.  5.0 is ONLY a change in the namespace, with no other changes.... at least few other changes... so few that if we do some they should be explicitly listed here.